README.md

```diff
@@ -85,6 +85,15 @@ Create a configurable option with one of the following names: `VCPU`, `Memory`,
 
 ![Plan traffic limit required options](./docs/plan-traffic-limit-required-options.png)
 
+#### Extra IPs
+
+The module gives the ability to select count of additional IPs on ordering of a product. You can manage the list of allowed values in the Configurable Options.
```

Comment:
The module gives the ability to select the count of additional IPs while ordering a product. You can manage the list of allowed values in the Configurable Options.